src/main/java/org/dependencytrack/integrations/defectdojo/DefectDojoClient.java

```diff
@@ -140,32 +171,40 @@ public ArrayList<String> jsonToList(final JSONArray jsonArray) {
     /*
      * A Reimport will reuse (overwrite) the existing test, instead of create a new test.
      * The Successfully reimport will also  increase the reimport counter by 1.
-    */
+     */
     public void reimportDependencyTrackFindings(final String token, final String engagementId, final InputStream findingsJson, final String testId) {
         LOGGER.debug("Re-reimport Dependency-Track findings to DefectDojo per Engagement");
-        final UnirestInstance ui = UnirestFactory.getUnirestInstance();
-        final HttpRequestWithBody request = ui.post(baseURL + "/api/v2/reimport-scan/");
-        final HttpResponse<String> response = request
-                .header("accept", "application/json")
-                .header("Authorization", "Token " + token)
-                .field("file", findingsJson, "findings.json")
-                .field("engagement", engagementId)
-                .field("scan_type", "Dependency Track Finding Packaging Format (FPF) Export")
-                .field("verified", "true")
-                .field("active", "true")
-                .field("minimum_severity", "Info")
-                .field("close_old_findings", "true")
-                .field("push_to_jira", "false")
-                .field("test", testId)
-                .field("scan_date", DATE_FORMAT.format(new Date()))
-                .asString();
-        if (response.getStatus() == 201) {
-            LOGGER.debug("Successfully reimport findings to DefectDojo");
-        } else {
-            LOGGER.warn("DefectDojo Client did not receive expected response while attempting to reimport"
-                    + "Dependency-Track findings. HTTP response code: "
-                    + response.getStatus() + " - " + response.getStatusText());
-            uploader.handleUnexpectedHttpResponse(LOGGER, request.getUrl(), response.getStatus(), response.getBody());
+        HttpPost request = new HttpPost(baseURL + "/api/v2/reimport-scan/");
+        request.addHeader("accept", "application/json");
+        request.addHeader("Authorization", "Token " + token);
+        List<NameValuePair> nameValuePairs = new ArrayList<>();
+        nameValuePairs.add(new BasicNameValuePair("engagement", engagementId));
```

Comment:
It seems nameValuePairs(header name and values) is repeated. Would it be better to extract it into a common method ?